edgraph/access_ee_test.go

```diff
@@ -1,113 +1,78 @@
 package edgraph
 
 import (
-	"fmt"
 	"strings"
 	"testing"
 	"time"
 
 	"github.com/dgraph-io/dgraph/ee/acl"
 	"github.com/dgraph-io/dgraph/worker"
-	jwt "github.com/dgrijalva/jwt-go"
-)
 
-var (
-	userDataList = []userData{
```

Comment:
I have moved this slice to TestValidateToken(), TestGetAccessJwt(), TestGetRefreshJwt().
Hope this was the expectation.